app/services/DynamoCampaigns.scala

```diff
@@ -0,0 +1,67 @@
+package services
+
+import com.typesafe.scalalogging.StrictLogging
+import models.Campaign
+import services.DynamoChannelTests.{DynamoDuplicateNameError, DynamoError, DynamoGetError, DynamoPutError}
```

Comment:
I think we should now move the `DynamoError` models into a separate file, because they're no longer only used by the `DynamoChannelTests` class.
How about creating `app/models/DynamoError.scala`?

Reply:
Good suggestion, thanks. I've decided to go with DynamoErrors as it contains several error classes.